gdal/doc/gdal_tutorial.dox

```diff
@@ -155,16 +154,19 @@ In C:
 
 In Python:
 \code
-    print 'Driver: ', dataset.GetDriver().ShortName,'/', \
-          dataset.GetDriver().LongName
-    print 'Size is ',dataset.RasterXSize,'x',dataset.RasterYSize, \
-          'x',dataset.RasterCount
-    print 'Projection is ',dataset.GetProjection()
+    print("Driver: {}/{}".format(dataset.GetDriver().ShortName,
+                                 dataset.GetDriver().LongName))
+
+    print("Size is {} x {} x {}".format(dataset.RasterXSize,
+                                        dataset.RasterYSize,
+                                        dataset.RasterCount))
+
+    print("Projection is {}".format(dataset.GetProjection()))
 
     geotransform = dataset.GetGeoTransform()
-    if not geotransform is None:
-	print 'Origin = (',geotransform[0], ',',geotransform[3],')'
-	print 'Pixel Size = (',geotransform[1], ',',geotransform[5],')'
+    if geotransform:
+        print("Origin = ({}, {})".format(geotransform[0], geotransform[3]))
+        print("Pixel Size = ({}, {})".format(geotransform[1], geotransform[5]))
 \endcode
```

Comment:
Ideally I wanted to use some descriptive names here, e.g. `print("Pixel Size = ({xsize}, {ysize})".format(xsize=geotransform[1], ysize=geotransform[5]))` but I was not sure what the elements of geotransform all really are so skipped that thought.